navigation:
    dist_to_marker: 0.5
    pos_thres: 0.2      # not used yet
    yaw_thres: 0.1      # not used yet
    vel_thres: 0.1      # not used yet
    vel_yaw_thres: 0.05 # not used yet

localization:
    process_noise: [0.005, 0.005, 0.005, 0.005, 0.005, 0.005]  # used in v5
    initial_cov: [100000, 100000, 100000, 100000, 100000, 100000] # used in v5
    maha_dist_thres: 3
    cov_norm_thres: 0.02
    measurement_config: [true, true, false,   # translation x,y,z
                         false, false, true] # rotation roll,pitch,yaw
    initial_map_to_odom: [1, 1, 0, 0, 0, 1.57] # [x,y,z,roll,pitch,yaw]

perception:
    img_size: [[480, 640], [446, 332]]  #H, W
    focal: [221.88078347, 220.72391163]   #Fx, Fy
    aruco_cov: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1] # used in marker_pose
    sign_cov: [0.1, 0.1, 0.1, -1, -1, -1] # not used in marker_pose


path_planning:
    rho: 0.15
    inflation: 0.1
